test/integration/host-mode-port-filter.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	stop_manager
+	stop_docker
+}
+
+@test "docker should filter port in host mode correctly" {
+	start_docker 3
```

Comment:
I think starting `2` should prove that it runs correctly (the 3rd one will fail), maybe there's no need to start 3.